.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -170,6 +170,9 @@ jobs:
             else
               export SHARD_NUMBER=0
             fi
+            if [[ -n $GITHUB_HEAD_REF && "$USE_CUDA" == 1 ]]; then
```

Comment:
This is the code that should trigger so that Windows CUDA would run smoke tests on PRs only.